modules/openy_features/openy_prgf/modules/openy_prgf_2c/openy_prgf_2c.info.yml

```diff
@@ -3,6 +3,7 @@ description: 'Implements a paragraph with 2 columns.'
 type: module
 core: 8.x
 package: OpenY
+version: '8.x-1.0'
 dependencies:
```

Comment:
The same question as above regarding modules:
- openy_block_flexible_content
- openy_block_custom_simple

Reply:
Fixed for `openy_block_flexible_content`, but we need at least 1 block in target columns, so `openy_block_custom_simple` will be required by openy profile and can't be decoupled